src/fidesops/main.py

```diff
@@ -29,21 +31,29 @@
     )
 
 app.include_router(api_router)
+for handler in ExceptionHandlers.get_handlers():
+    app.add_exception_handler(FunctionalityNotConfigured, handler)
 
 
 def start_webserver() -> None:
     """Run any pending DB migrations and start the webserver."""
     logger.info("****************fidesops****************")
-    logger.info("Running any pending DB migrations...")
-    init_db(config.database.SQLALCHEMY_DATABASE_URI)
+    if config.database.ENABLED:
+        # don't run db migrations if database is disabled
+        logger.info("Running any pending DB migrations...")
+        init_db(config.database.SQLALCHEMY_DATABASE_URI)
+
     scheduler.start()
 
-    logger.info("Starting scheduled request intake...")
-    initiate_scheduled_request_intake()
+    if config.database.ENABLED:
+        # don't schedule request intake if database is disabled
+        logger.info("Starting scheduled request intake...")
+        initiate_scheduled_request_intake()
 
     logger.info("Starting web server...")
+
     uvicorn.run(
-        "src.fidesops.main:app",
```

Comment:
when `init_db` wasn't run, uvicorn wasn't finding the `fidesops` module anymore when this was the run command. i think i was able to pin it down to the fact that within `init_db()`, we run `src/migrations/env.py`, which happens to add `..` to the `sys.path` -- and only because of that the `src.fidesops` path is picked up correctly. to me this seems like an unrecognized and unintentional side-effect -- `init_db()` should not be needed to get the base app running, i don't think -- if it is intended, then we need to do it in a much less obfuscated way.

as far as i could tell, making this switch here doesn't impact existing behavior, and it also allows the app to run if `init_db` is not called. i don't know of any negative consequences (though i could certainly be missing something!)

Reply:
You explanation here makes sense to me and everything seems to work fine with your change.